docs/en/observability/infrastructure-threshold-alert.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[infrastructure-threshold-alert]]
+= Create an infrastructure threshold alert
```

Comment:
I didn't have time to test this topic, but it looks pretty straightforward, so I think it's OK. I wonder, though, if these topics about creating alerts should be more procedural (actual steps). If I come to this page from a google search, I might not understand how to get to the Inventory page and what I select to create an alert. Anyhow, it's up to you. I just think the steps don't pop like I would want them to.